scripts/gen-universe.py

```diff
@@ -59,18 +59,32 @@ def main():
     create_content_type_file(ct_universe_path, "v4")
 
     # Render empty json
-    empty_path = args.outdir / 'repo-empty-v3.json'
-    with empty_path.open('w', encoding='utf-8') as universe_file:
-        json.dump({'packages': []}, universe_file)
-    ct_empty_path = args.outdir / 'repo-empty-v3.content_type'
-    create_content_type_file(ct_empty_path, "v3")
+    render_empty_json(args.outdir, "v3")
+    render_empty_json(args.outdir, "v4")
 
     # create universe-by-version files for `dcos_versions`
     dcos_versions = ["1.6.1", "1.7", "1.8", "1.9", "1.10"]
     [render_universe_by_version(
         args.outdir, packages, version) for version in dcos_versions]
 
 
+def render_empty_json(outdir, universe_version):
+    """ Renders an empty universe json of with version `universe_version`
+
+    :param outdir: Path to the directory to use to store all universe objects
+    :type outdir: str
+    :param universe_version: Universe content type version: "v3" or "v4"
+    :type universe_version: str
+    :rtype: None
+    """
+    base_name = 'repo-empty-{}'.format(universe_version)
+    empty_path = outdir / (base_name + ".json")
```

Comment:
Why not include the `.json` in the string on the previous line? It should be easier to read.